ports/qt/vcpkg.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "qt",
+  "version-string": "6.0.2",
+  "description": "Qt",
+  "homepage": "https://www.qt.io/",
+  "dependencies": [
+    "qt5compat",
+    "qtbase",
+    "qtcoap",
+    "qtdeclarative",
+    "qtdoc",
+    "qtimageformats",
+    "qtmqtt",
+    "qtnetworkauth",
+    "qtopcua",
+    "qtquick3d",
+    "qtquickcontrols2",
+    "qtquicktimeline",
+    "qtshadertools",
+    "qtsvg",
+    "qttools",
+    "qttranslations"
+  ],
+  "default-features": [
+    "default-features"
+  ],
+  "features": {
+    "default-features": {
+      "description": "Platform-dependent default features",
+      "dependencies": [
+        {
+          "name": "qtwayland",
+          "platform": "!windows"
```

```suggestion
          "platform": "linux"
```
Currently it wants to install qtwayland on my mac.